azplugins/BondEvaluatorFENE.h

```diff
@@ -108,7 +108,7 @@ class BondEvaluatorFENE
         DEVICE bool evalForceAndEnergy(Scalar& force_divr, Scalar& bond_eng)
             {
             // check for invalid parameters
-            if (lj1 == Scalar(0.0) || r_0 == Scalar(0.0) || K == Scalar(0.0) ) return false;
+            if (r_0 == Scalar(0.0) || K == Scalar(0.0) ) return false;
```

Comment:
This is right, but I am actually not sure why we are validating parameters inside the kernel here. Let's leave the check on `r_0` since it prevents divide-by-zero, but I don't see a problem with `K == 0` either:
```suggestion
            if (r_0 == Scalar(0.0)) return false;
```